docs/data/rpc/admin-guide/installing.mdx

```diff
@@ -0,0 +1,168 @@
+---
+title: Installing
+sidebar_position: 20
+---
+
+We recommend the following ways to deploy your own RPC instance:
+
+1. Deploy to Kubernetes using [Helm](https://helm.sh/docs/intro/install/)
+2. Run the [stellar-rpc docker image](https://hub.docker.com/r/stellar/stellar-rpc) directly
```

Comment:
I think we need to add instructions for the following methods:
- Debian Packages
- Building from Source 

I also think we should consider removing the helm option from these docs. We don't dogfood those helm charts internally so I'm not convinced they work. Even if they do, it's likely they'll become out-of-date over time. That said, if we want to keep this installation method in the docs, then we should at least sanity-check that these steps work as part of this docs cleanup.

Reply:
yes, `Kubernetes With Helm` section should be removed for now as it's not working with latest versions. 
I tried deployment on sandbox to verify and it didn't work. Looks like there is an error on the core cfg templates from the chart - filed issue [helm-charts/113](https://github.com/stellar/helm-charts/issues/113)